### ISSUE TITLE
Discord?

### ISSUE THREAD

**Issue by corysimmons** [nandorojo/solito#5]
It'd be cool to have a Discord for the community around this stack.

**Comment by nandorojo** [comment]
yeah i agree, it’s something i still have to give some thought to

**Comment by Rychillie** [comment]
Any news on a project Discord?

**Comment by nandorojo** [comment]
Better late than never. Discord created here: https://discord.gg/JtvAVzNpDk
